lib/pos_mempool.go

```diff
@@ -116,12 +115,6 @@ func GetAugmentedUniversalViewWithAdditionalTransactions(
 	return newView, nil
 }
 
-type MempoolIterator interface {
```

Comment:
The public MempoolIterator is unused. At this point, it is no longer thread-safe because the memool's transaction register can be changed from underneath.